src/components/Lesson/LessonList/index.tsx

```diff
@@ -1,43 +1,48 @@
-import React, { FC } from 'react';
-import { FlatList, TouchableOpacity } from 'react-native-gesture-handler';
+import React, { FC, memo } from 'react';
+import { FlatList } from 'react-native-gesture-handler';
 import { HorizontalLessonCard } from '../HorizontalLessonCard';
 import { VerticalLessonCard } from '../VerticalLessonCard';
 import useConnect from './connect';
 import { HorizontalView, VerticalView } from './styles';
 import { Props } from './types';
 
-export const LessonList: FC<Props> = ({ categoryFlag }) => {
+export const LessonList: FC<Props> = ({ isFiltered, lessons }) => {
   const { handleNavigateToDetail } = useConnect();
+  const categoryFlag = 'All'.localeCompare(isFiltered) === 0;
   return (
     <>
       {categoryFlag ? (
-        <HorizontalView>
+        <VerticalView>
           <FlatList
-            key={'1colum'}
-            data={['lesson1', 'lesso2', 'lesson3', '4', '5', '6']}
+            key={'2colums'}
```

Comment:
`key` is not correct here and it should never be a hardcoded string. Key is something dynamic.

Remember the explanation I left on our Slack conversation and use `keyExtractor`, check this: https://github.com/josgarmar28/z1-react-native/pull/6

Reply:
Thanks, It has been fixed following the same method as you suggested in the link